deploy.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Build the contract
+mox build
+
+# Load environment variables from .env file
+if [ -f .env ]; then
+    export $(cat .env | grep -v '^#' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)